consensus/integration/integration_test.go

```diff
@@ -72,7 +72,7 @@ func Test5Nodes(t *testing.T) {
 
 	runNodes(signalerCtx, nodes)
 
-	<-stopper.stopped
+	unittest.RequireCloseBefore(t, stopper.stopped, 30*time.Second, "expect to stop before timeout")
```

Comment:
unfortunattely this test timed out on CI 😭 
https://github.com/onflow/flow-go/runs/8029489368?check_suite_focus=true

Reply:
`Test2TimeoutOutof4Instances` seems to have as well :(

Reply:
Should be fixed by https://github.com/onflow/flow-go/pull/2935/commits/4c603db291f2fd0aa82cf8a0da17330b4e6e1b5c 🤞